src/modules/api.js

```diff
@@ -0,0 +1,47 @@
+const apiKey = 'xOhJaij0iL0K3CoP6gQZ';

```

Comment:
- [ ] Kindly fix linter errors because i can see that your linters are not passing all the tests 🤔